plans/super/169-row-count-between.md

```
@@ -0,0 +1,648 @@
+
+**Acceptance criteria.** Rule file accurately reflects the 2-instance precedent. Memory captures the dispatch-arm list + sqlglot reuse pattern.
+
+**Done when.** Plan-doc owner reviews the rule update and approves.
+
+**Depends on:** US-013.
+
+---
+
+### Story dependency graph
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)